Style test is not passing

<!--
This is a suggested issue template for tedana.

If there is other information that would be helpful to include, please do not hesitate to add it!

Before submitting, please check to make sure that the issue is not already addressed; if there is a related issue, then please cross-reference it by #.
If this is a usage question, please check out NeuroStars here:
https://neurostars.org/
and tag your topic with "tedana"
-->

<!--
Summarize the issue in 1-2 sentences, linking other issues if they are relevant

Note: simply typing # will prompt you for open issues to select from
-->
### Summary

The style tests are not passing, showing the error below.

I don't know if something changed with `black` or we somehow missed these.

<!--
If needed, add additional detail for:
1. Recreating a bug/problem
2. Any additional context necessary to understand the issue
-->
### Additional Detail

```
tedana/selection/selection_utils.py:46:9: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
tedana/selection/selection_utils.py:48:9: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
tedana/selection/selection_utils.py:53:11: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
tedana/selection/selection_utils.py:60:11: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
tedana/workflows/ica_reclassify.py:215:10: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
tedana/workflows/ica_reclassify.py:218:10: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
make: *** [Makefile:16: lint] Error 1

Exited with code exit status 2
```

<!--
If desired, add suggested next steps.
If you foresee them in a particular order or priority, please use numbering
-->
### Next Steps

<!-- 
Thank you for submitting your issue!
If you do not receive a response within a calendar week, please post a comment on this issue to catch our attention. 
Some issues may not be resolved right away due to the volunteer nature of the project; thank you for your patience!
-->

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -40,9 +40,9 @@ Paper = "https://joss.theoj.org/papers/10.21105/joss.03669"
 
 [project.optional-dependencies]
 doc = [
-    "sphinx>=1.5.3",
+    "sphinx>=6.2.1",
     "sphinx_copybutton",
-    "sphinx_rtd_theme",
+    "sphinx_rtd_theme>=1.2.2",
     "sphinx-argparse",
     "sphinxcontrib-bibtex",
 ]

--- a/tedana/selection/selection_utils.py
+++ b/tedana/selection/selection_utils.py
@@ -43,21 +43,21 @@ def selectcomps2use(selector, decide_comps):
             "selector.component_table needs a 'classification' column to run selectcomp2suse"
         )
 
-    if (type(decide_comps) == str) or (type(decide_comps) == int):
+    if type(decide_comps) in [str, int]:
         decide_comps = [decide_comps]
-    if (type(decide_comps) == list) and (decide_comps[0] == "all"):
+    if (type(decide_comps) is list) and (decide_comps[0] == "all"):
         # All components with any string in the classification field
         # are set to True
         comps2use = list(range(selector.component_table.shape[0]))
 
-    elif (type(decide_comps) == list) and all(isinstance(elem, str) for elem in decide_comps):
+    elif (type(decide_comps) is list) and all(isinstance(elem, str) for elem in decide_comps):
         comps2use = []
         for didx in range(len(decide_comps)):
             newcomps2use = selector.component_table.index[
                 selector.component_table["classification"] == decide_comps[didx]
             ].tolist()
             comps2use = list(set(comps2use + newcomps2use))
-    elif (type(decide_comps) == list) and all(type(elem) == int for elem in decide_comps):
+    elif (type(decide_comps) is list) and all(type(elem) is int for elem in decide_comps):
         # decide_comps is already a string of indices
         if len(selector.component_table) <= max(decide_comps):
             raise ValueError(

--- a/tedana/workflows/ica_reclassify.py
+++ b/tedana/workflows/ica_reclassify.py
@@ -212,10 +212,10 @@ def _parse_manual_list(manual_list):
     elif op.exists(op.expanduser(str(manual_list[0]).strip(" "))):
         # filename was given
         manual_nums = fname_to_component_list(op.expanduser(str(manual_list[0]).strip(" ")))
-    elif type(manual_list[0]) == str:
+    elif type(manual_list[0]) is str:
         # arbitrary string was given, length of list is 1
         manual_nums = str_to_component_list(manual_list[0])
-    elif type(manual_list[0]) == int:
+    elif type(manual_list[0]) is int:
         # Is a single integer and should remain a list with a single integer
         manual_nums = manual_list
     else:

